src/librustc/lint/mod.rs

```diff
@@ -27,7 +27,9 @@ use crate::hir::def_id::{CrateNum, LOCAL_CRATE};
 use crate::hir::intravisit;
 use crate::hir;
 use crate::lint::builtin::BuiltinLintDiagnostics;
-use crate::lint::builtin::parser::{QUESTION_MARK_MACRO_SEP, ILL_FORMED_ATTRIBUTE_INPUT};
+use crate::lint::builtin::parser::{
+    ILL_FORMED_ATTRIBUTE_INPUT, META_VARIABLE_MISUSE, QUESTION_MARK_MACRO_SEP
+};
```

Comment:
Style nit: please break into multiple `use`s if imports stop fitting into a single line.

Reply:
This doesn't apply anymore after merge. It fits on one line. But that's good to know, thanks.